Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby file: ".ruby-version"
+ruby "3.2.3"
```

Comment:
Is there an issue with picking up the version from the `.ruby-version` file?

Reply:
GAH